commands/cli.go

```diff
@@ -201,7 +217,7 @@ func awsNuke(c *cli.Context) error {
 	}
 
 	logging.Logger.Infof("Retrieving active AWS resources in [%s]", strings.Join(targetRegions[:], ", "))
-	account, err := aws.GetAllResources(targetRegions, *excludeAfter, resourceTypes)
+	account, err := aws.GetAllResources(targetRegions, *excludeAfter, resourceTypes, *configObj)
```

Comment:
Couldn't `configObj` be `nil` at this point if the user didn't specify one? If so, will this be a nil pointer dereference?

Reply:
Suggestion for fix in the initial assignment:

```
	configObj := config.Config{}
	configFilePath := c.String("config")

	if configFilePath != "" {
		configObjPtr, err = config.GetConfig(configFilePath)

		if err != nil {
			return fmt.Errorf("Error reading config - %s - %s", configFilePath, err)
		}
                configObj = *configObjPtr
	}
```

Reply:
- [ ] TODO